cypress/e2e/shared/tasks.test.ts

```diff
@@ -414,7 +410,8 @@ from(bucket: "defbuck")
     })
 
     it('can click to filter tasks by labels', () => {
-      cy.getByTestID('task-card').should('have.length', 2)
+      // should be > 1 length
+      cy.getByTestID('task-card').should('not.have.length', 1)
```

Comment:
This comment and this line can probably be combined into a more self-documenting statement:

```ts
cy.getByTestID('task-card').should('have.length.greaterThan', 1)
```

Reply:
excellent point. done.